sdk/src/types/block/slot/index.rs

```diff
@@ -17,6 +17,11 @@ impl SlotIndex {
     pub fn new(index: u64) -> Self {
         Self::from(index)
     }
+
+    /// Returns the inner value.
+    pub fn get(&self) -> u64 {
```

Comment:
impl deref ;) or make it `pub struct SlotIndex(pub u64);`

Reply:
It already derives Deref